testapi.pm

```diff
@@ -1806,6 +1806,14 @@ sub diag {
     return bmwqemu::diag(@_);
 }
 
+=head2 get_host_ip
+    Return VM's host IP
+    in a kvm instance you reach the VM's host under 10.0.2.2
+=cut
+sub get_host_ip {
```

Comment:
as you have no set_ function, the get_ prefix is just noise.

Reply:
Fixed